README.md

```diff
@@ -1,18 +1,24 @@
 PGObserver
 ==========
 
-PGObserver was developed at Zalando during the past years to monitor performance metrics of our different PostgreSQL clusters. Due to our use of a stored procedure API layer our strong focus was on monitoring that API layer. However the number of metrics was increased to include other relevant values:
+PGObserver is a monitoring solution for PostgreSQL databases, covering almost all metrics provided by the database engine's internal statistics collector.
+PGObserver works out of the box on all Postgres databases from version 9.0 (including AWS RDS) and does not require installing any server side database extensions nor privileged users, making it perfect for developers!
+For some metrics though, data gathering wrapper functions (stored procedures) need to be installed on the server being monitored, to circumvent the superuser requirements.
 
-* Stored procedure data (pure sql procedures are not tracked)
+Monitored metrics include:
+
+* Stored procedure data
     - number of calls
     - run time per procedure
     - self time per procedure
+* All executed statements data (based on pg_stat_statements module, which needs to be enabled on the DB)
+    - query runtimes, call counts, time spent on IO
 * Table IO statistics
     - number of sequential and index scans
     - number of inserts, (hot) updates, deletes
     - table and index size
     - heap hits vs disk hits
-* CPU load
+* CPU load (needs a plpythonu stored procedure from sql/data_collection_helpers folder)
```

Comment:
this is the second "(requires/needs ...)"; maybe you should have a bullet point "*Requirement - blablabla..." to clearly structure these clauses